web/embed_components.html

```diff
@@ -52,7 +81,7 @@
   </style>
   <template>
     <div>
-      <paper-tabs selected="0" class="tabs bottom fit vertical">
+      <paper-tabs class="tabs" selected="0" style="position:relative" class="bottom fit vertical" style="top: inherit">
```

Comment:
Polymer has a `relative` class selector you can use instead of using an inline style.

You have two `class` attributes here - you can combine them.